packages/itwin/grouping-mapping-widget/package.json

```diff
@@ -102,6 +107,7 @@
     "react-table": "^7.7.0",
     "redux": "^4.1.0",
     "rimraf": "^3.0.2",
+    "split.js": "^1.6.0",
```

Comment:
Why is split.js a devDependency? I see that react-split has a dependency on split.js, but I don't understand why we need to explicitly add it into our package.json.

Reply:
This wasn't suppose to be here. I used it to hoist up the package to check the types when developing this. I'll let you know when this is review ready. I am taking my laptop to another location and I pushed what I had just incase something happens to it.

Reply:
Removed